notebooks/case1_bike_count.md

```diff
@@ -130,67 +140,45 @@ Pre-process the data:
 ```{code-cell} ipython3
 :tags: [nbtutor-solution]
 
-combined = df['datum'] + ' ' + df['tijd']
-combined.head()
+df["timestamp"] = pd.to_datetime(df["Ordening"], format="%Y-%m-%dT%H:%M:%S%z", utc=True)
 ```
 
 ```{code-cell} ipython3
 :tags: [nbtutor-solution]
 
-df.index = pd.to_datetime(combined, format="%d/%m/%Y %H:%M")
+df = df.set_index("timestamp")
 ```
 
 ```{code-cell} ipython3
 :tags: [nbtutor-solution]
 
-df2 = df.drop(columns=['datum', 'tijd'])
+df2022 = df.drop(columns=['Datum', 'Uur5Minuten', 'Ordening', 'Code'])
 ```
 
 ```{code-cell} ipython3
 :tags: [nbtutor-solution]
 
-df2 = df2.rename(columns={'ri Centrum': 'direction_centre',
-                          'ri Mariakerke':'direction_mariakerke'})
+df2022 = df2022.rename(columns={'Tegenrichting': 'direction_centre',
+                          'Hoofdrichting': 'direction_mariakerke',
+                          'Totaal': 'total',
+                          'Locatie': 'location'
+                         })
 ```
 
 ```{code-cell} ipython3
-df2.head()
+df2022.head()
 ```
 
 Having the data available with an interpreted `datetime`, provides us the possibility of having time aware plotting:
 
 ```{code-cell} ipython3
 fig, ax = plt.subplots(figsize=(10, 6))
-df.plot(colormap='coolwarm', ax=ax)
-```
-
-<div class="alert alert-warning">
-
- <b>Remark</b>: Interpretation of the dates with and without predefined date format.
```

Comment:
FWIW it's also fine that we remove this section anyway, because the latest version of pandas will guess a format when you don't specify one, and so you don't see this performance difference anymore (unless you explicitly set `format="mixed"`)